src/util/RouterUtil.js

```diff
@@ -133,17 +133,12 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
 
       var redirectFn = router.settings.get('redirectUtilFn');
 
-      if (res.type === Enums.SESSION_STEP_UP) {
-        var targetUrl = res._links && res._links.next && res._links.next.href;
-        successData.stepUp = {
-          url: targetUrl,
-          finish: function () {
-            redirectFn(targetUrl);
-          }
+      var nextUrl = res._links && res._links.next && res._links.next.href;
+      if (nextUrl) {
+        successData.next = function () {
```

Comment:
I opted for `.next` instead of `finish`, and since we don't use `stepUp.url` anywere in okta-core, I decided to totally remove the wrapping object instead of just renaming it. What do you think about this? Feel free to propose better names/structure.

Reply:
that's completely fine

Reply:
shall we override the `type` to `Enums.SESSION_SSO` as same as the `else` block rather than at line 127?

Reply:
You'll actually need to update the README since it calls out these values

Reply:
I agree with you that we should update the README, but we cannot do it in this PR and until we move to use only the new way (see discussion with @tthompson-okta in the other comment where we decided to keep it backward compatible), because right now to ensure retrocompatibility, the old way (`.stepUp.finish()`) needs to have still priority on the new one in the if conditions. If we update README and developers start to use `.next`, they will still receive the response from SESSION_STEP_UP with the `stepUp.finish` function and not the new `.next`, so their code would not work.